src/main/java/Duke.java

```diff
@@ -1,10 +1,59 @@
+import java.util.Scanner;
 public class Duke {
+    Tasks taskList[] = new Tasks[100];
+    void printWelcome() {
+        System.out.println("Hello I'm Duke\nWhat can I do for you?");
+    }
+    void acceptCommands() {
+        Scanner input = new Scanner(System.in);
+        String command;
+        int noOfCommands = 0;
+        do {
+            command = input.nextLine();
+            if(command.equals("list")) {
+                displayList(noOfCommands);
+                continue;
+            }
+            if(command.indexOf(' ') > 0 && command.substring(0,command.indexOf(' ')).equals("mark")) {
```

Comment:
Try to add a variable to it. Reading the statement can be difficult especially when the condition is long.